### ISSUE TITLE
Google Analytics route change handling mistake

### ISSUE THREAD

**Issue by snegostup** [gpbl/isomorphic500#62]
Hi,
You should also send path with GA 'pageview' request as stated here https://developers.google.com/analytics/devguides/collection/analyticsjs/single-page-applications otherwise all subsequent hits are counted for the entry page.

So this https://github.com/gpbl/isomorphic500/blob/c68b41b3dbb791b139816819bc500a43f8819efb/src/utils/trackPageView.js#L4-L10 should become 
```javascript
function trackPageView(path) {
    if (!window.ga) {
        return;
    }
    window.ga("send", "pageview", path);
}
```

And this https://github.com/gpbl/isomorphic500/blob/c68b41b3dbb791b139816819bc500a43f8819efb/src/Application.js#L43-L45 should become

```javascript
if (!Immutable.is(prevProps.currentRoute, currentRoute)) {
    trackPageView(currentRoute.get('url'));
}
```

**Comment by gpbl** [comment]
Oh yes! Thanks a lot for spotting this. Care to make a PR or should I work on it?

**Comment by gpbl** [comment]
Fixed with #63
